### PR TITLE
[11.0] Fix CI build adding postgis scripts

### PR DESCRIPTION
**Pull request by yvaucher**
Postgis scripts are included in travis build.
However, runbot doesn't pre-install them. Thus it needs to be set
explicitely.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,34 +14,30 @@ addons:
...
-
 env:
-  matrix:
-  - VERSION="11.0" LINT_CHECK="1"
-  - VERSION="11.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - VERSION="11.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
   global:
-    secure: "HLfds+sQG4JcM53.5b/sTemGvBlVNdfGc4XvJ+py2rx25F4BWojpbOYx44nSiUjBO9U6Eo5kwiK0mWEXoPGg6trQcbiur9kgHrV3+hydlUUKe2/P/mnoP2JAxtu0JBSLxNcG40FdIwEE3Sd0w/ocmxMOp3RmOOVacAY48eWYeh8="
+    - VERSION="11.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+    - secure: "HLfds+sQG4JcM52D7b/sTemGvBlVNdfGc4XvJ+py2rx25F4BWojpbOYx44nSiUjBO9U6Eo5kwiK0mWEXoPGg6trQcbiur9kgHrV3+hydlUUKe2/P/mnoP2JAxtu0JBSLxNcG40FdIwEE3Sd0w/ocmxMOp3RmOOVacAY48eWYeh8="
```
Comment (pedrobaeza) [inline .travis.yml]: This is not needed anymore

Comment (yvaucher) [reply .travis.yml]: What was it for already? Transifex?

Comment (pedrobaeza) [reply .travis.yml]: That's it!

Comment (yvaucher) [reply .travis.yml]: just removed that line.

**Comment by pedrobaeza** [comment]
A little pylint issue:
```
base_geoengine/README.rst:29: [E7901(rst-syntax-error), ]  Duplicate implicit target name: "geospatial support for odoo".
```

**Comment by yvaucher** [comment]
Back ported few fixes from https://github.com/OCA/geospatial/pull/185

**Pull request merged**